software/memex.yml

```diff
@@ -0,0 +1,10 @@
+name: "memEx"
+website_url: "https://gitea.bubbletea.dev/shibao/memEx"
+source_code_url: "https://gitea.bubbletea.dev/shibao/memEx"
+description: "A structured, zettlekasten and org-mode inspired memex."
```

Comment:
This could be slightly tweaked to explain what a `memex` is in plain English so that people looking for a project but might not know what it is, can be interested.